### PR TITLE
[metricbeats][filebeats] Ran doctoc to update FAQ which updated sub-folders as well.

### PR DESCRIPTION
**Pull request by kwsorensen**
- [ x] Chart version *not* bumped (the versions are all bumped and released at the same time)
- [x ] README.md updated with any new values or changes
- [ ] Updated template tests in `${CHART}/tests/*.py` 
- [ ] Updated integration tests in `${CHART}/examples/*/test/goss.yaml`

I just wanted to see my FAQ updated in the README's TOC

### CODE REVIEW

**Comment by kwsorensen** [comment]
Ran the `docker run --rm -it -v $(pwd):/usr/src jorgeandrada/doctoc` according to the readme to update the TOC. To represent my previous addition to the READMEs

**Comment by kwsorensen** [comment]
Follow up to https://github.com/elastic/helm-charts/pull/997.

File: filebeat/README.md [hunk 1]
```diff
@@ -16,12 +16,12 @@ This Helm chart is a lightweight way to configure and run our official
   - [Install development version using master branch](#install-development-version-using-master-branch)
 - [Upgrading](#upgrading)
 - [Usage notes](#usage-notes)
-- [Configuration](#configuration)
```
Comment (jmlrt) [inline filebeat/README.md:19]: Tghis shouldn't be removed from the TOC. It seems this header was removed by mistake in a previous PR.

File: filebeat/README.md [hunk 2]
```diff
@@ -190,6 +190,14 @@ namespace which gives it access to the host loopback device, services listening
 on localhost, could be used to snoop on network activity of other pods on the
 same node.
 
+### How do I get multiple beats agents working with hostNetworking enabled?
```
Comment (jmlrt) [inline filebeat/README.md:193]: This shouldn't be seen as a new change as #997 has been merged. Can you try to rebase master on your branch?

File: metricbeat/README.md [hunk 3]
```diff
@@ -204,6 +205,14 @@ namespace which gives it access to the host loopback device, services listening
 on localhost, could be used to snoop on network activity of other pods on the
 same node.
 
+### How do I get multiple beats agents working with hostNetworking enabled?
```
Comment (jmlrt) [inline metricbeat/README.md:208]: This shouldn't be seen as a new change as #997 has been merged. Can you try to rebase master on your branch?

**Review by jmlrt — CHANGES_REQUESTED** [review]
Hi @kwsorensen,
Thanks for this PR.

- Can you rebase your branch on master?
- Also README files in examples directory shouldn't be updated with TOC. Can you remove this change?
- Finally It seems tha `Configuration` header has been dropped from Filebeat README by mistake in https://github.com/elastic/helm-charts/pull/964/files#diff-9148fe0a90c1405cb044332ee4f53092291d5c4c6a24d3d7d999d8fa056ab5e1L89. It could be great if you can fix it in this PR. (I can fix it in another PR otherwise).

**Pull request closed without merging**